grazer/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -57,58 +58,97 @@
 	</c:if>
   </head>
 
-  <body>
-  <div style="opacity: 0.1; position: absolute; top: 10px; left: 10px; ">
-    <img src="<c:url value="/resources/images/ep-logo.gif" />" />
-  </div>
-    <div class="container" style="padding-bottom: 150px;">
+  <body data-spy="scroll" data-offset="0" data-target="#navigation">
+    <div id="navigation" class="navbar navbar-default">
+      <div class="container-fluid">
+          <sec:authorize access="isAuthenticated()">
+            <div class="container">
+              <div class="row" style="padding-top: 15px;">
+                Welcome <i class="fa fa-user"></i><span style="margin-left: 5px;"><sec:authentication property="principal.username" /></span>!
+                <form action="<c:url value='/logout' />" method='POST' class="pull-right">
+                  <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
+                  <div>
+                    <button type="submit" class="btn btn-link" style="color:#800000"><i class="fa fa-sign-out" aria-hidden="true"></i> Logout</button>
+                  </div>
+                </form>
+                <button type="button" class="btn btn-link pull-right"><a href="${pageContext.servletContext.contextPath}/api/transformations/start">Start Data Import</a></button>
```

Comment:
let's also show a warning dialog after the user clicks that button that warns them that when they proceed previously imported data will be deleted and asks if they want to proceed.